SoundCloud/Fastfile

```diff
@@ -0,0 +1,195 @@
+  end
+
+  ##### Apple Member Center ######
+  desc "Update the devices with the ones in devices.txt"
+  lane :update_devices do
+    register_devices(devices_file: 'devices.txt', team_id: TEAM_ID)
+  end
+
+  desc "Fetches the development certificates and provisioning profiles to run the project on real devices"
+  lane :dev_certificates_fetch do
+    match(type: "development", app_identifier: SoundCloud::Models::Project.new("Debug").app_bundle_id, readonly: true)
+    match(type: "development", app_identifier: SoundCloud::Models::Project.new("Debug").watchapp_bundle_id, readonly: true)
+    match(type: "development", app_identifier: SoundCloud::Models::Project.new("Debug").watchextension_bundle_id, readonly: true)
+  end
+
+  desc "Fetches the distribution certificates and provisioning profiles compile the app for AppStore distribution"
+  lane :dist_certificates_fetch do
+    match(type: "appstore", app_identifier: SoundCloud::Models::Project.new("AppStore").app_bundle_id, readonly: true)
+    match(type: "appstore", app_identifier: SoundCloud::Models::Project.new("AppStore").watchapp_bundle_id, readonly: true)
+    match(type: "appstore", app_identifier: SoundCloud::Models::Project.new("AppStore").watchextension_bundle_id, readonly: true)
+  end
+
+  desc "Updates the development provisioning profiles"
+  lane :dev_certificates_update do
+    match(type: "development", app_identifier: SoundCloud::Models::Project.new("Debug").app_bundle_id, team_id: TEAM_ID, username: USERNAME, readonly: false, force: true)
+    match(type: "development", app_identifier: SoundCloud::Models::Project.new("Debug").watchapp_bundle_id, team_id: TEAM_ID, username: USERNAME, readonly: false, force: true)
+    match(type: "development", app_identifier: SoundCloud::Models::Project.new("Debug").watchextension_bundle_id, team_id: TEAM_ID, username: USERNAME, readonly: false, force: true)
+  end
+
+  desc "Updates the distribution provisioning profiles"
+  lane :dist_certificates_update do
+    match(type: "appstore", app_identifier: SoundCloud::Models::Project.new("AppStore").app_bundle_id, team_id: TEAM_ID, username: USERNAME, readonly: false, force: true)
+    match(type: "appstore", app_identifier: SoundCloud::Models::Project.new("AppStore").watchapp_bundle_id, team_id: TEAM_ID, username: USERNAME, readonly: false, force: true)
+    match(type: "appstore", app_identifier: SoundCloud::Models::Project.new("AppStore").watchextension_bundle_id, team_id: TEAM_ID, username: USERNAME, readonly: false, force: true)
+  end
+
+  # Test
+  desc "Runs unit tests"
+  lane :unit_tests do |params|
+    device = params[:device].to_s == '' ? "iPhone 6 (9.1)" : params[:device]
+    scan(output_types: "html, junit", device: device)
+  end
+
+  desc "Run acceptance tests"
+  lane :acceptance_tests do
+    build_frank
+    sh "rake spec:cucumber"
+  end
+
+  # Linting
+  desc "Executes Danger"
+  lane :danger do
+    sh "script/danger"
+  end
+
+  # Ci
+  lane :ci do |params|
```

Comment:
There's actually no reason. We did that way instead. Do you think it would make more sense to split it instead?